provision/rosa-cross-dc/keycloak-benchmark-crossdc-tests/src/test/java/org/keycloak/benchmark/crossdc/DatacenterConfig.java

```diff
@@ -0,0 +1,63 @@
+package org.keycloak.benchmark.crossdc;
+
+public class DatacenterConfig {
+
+    private static String REALM;
+    private static String KEYCLOAK_SERVER_URL;
+    private static String ISPN_REST_ENDPOINT;
+    private static String BASE_URL;
+    private static String LOGIN_ENDPOINT;
+    private static String LOGOUT_ENDPOINT;
+    private static String TOKEN_ENDPOINT;
+    private static String REDIRECT_URI;
+    private static String DATASET_STATUS_URL;
+    private static String DATASET_LAST_CLIENT_URL;
+    private static String DATASET_LAST_USER_URL;
+    public static void setKeycloakDCUrls(String keycloakServerUrl, String infinispanRestEndpointUrl) {
```

Comment:
Might be better to just have classic objects initialized once via constructor instead of constantly resetting this static stuff throughout the test. Initialize the objects in the setup method and then just use dc1.getLoginEndpoint(), dc2.getISPNEnpoint(), etc. from the test method.

Reply:
@tkyjovsk I was thinking about that as well, this was more quick and dirty to create some abstraction, what you were thinking would that be something along these lines ?

```
private final String REALM;
private final String KEYCLOAK_SERVER_URL;
private final String ISPN_REST_ENDPOINT;
private final String BASE_URL;
private final String LOGIN_ENDPOINT;

public DatacenterConfig2(String keycloakServerUrl, String infinispanRestEndpointUrl) {
        this.KEYCLOAK_SERVER_URL = keycloakServerUrl;
        this.ISPN_REST_ENDPOINT = infinispanRestEndpointUrl;
        this.REALM =  "realm-0";
        this.BASE_URL = KEYCLOAK_SERVER_URL + "/realms/" + REALM;
        this.LOGIN_ENDPOINT = BASE_URL + "/protocol/openid-connect/auth";
        }
        
    public String getKEYCLOAK_SERVER_URL() {
        return KEYCLOAK_SERVER_URL;
    };
    public String getISPN_REST_ENDPOINT() {
        return ISPN_REST_ENDPOINT;
    };
    public String getREALM(){
        return REALM;
    };
    public String getLOGIN_ENDPOINT() {
        return LOGIN_ENDPOINT;
    };
```

Reply:
this might need little bit more thinking given how some of the current methods are written, I would consider this as an near term improvement. @tkyjovsk feel free to commit what solution you have in mind, if you have the time or we can tackle it in a separate PR.

Reply:
@kami619 @martin-kanis Commited the solution I had in mind. The private utility methods now also take `DatacenterInfo` argument to specify which DC they apply to. Check out the latest commit and lmk if that's ok. Hopefully I didn't break anything.

Reply:
Thats great, I just tested it and it works. I see you also simplified the urls, appreciate the extra mile you went to push the solution you had in mind. @tkyjovsk